Workflow failure: weekly-pulumi-update

## Workflow Failure

[weekly-pulumi-update](https://github.com/pulumi/pulumi-kubernetes-operator/blob/master/.github/workflows/weekly-pulumi-update.yml) has failed. See the list of failures below:

- [2025-02-27T12:50:31.000Z](https://github.com/pulumi/pulumi-kubernetes-operator/actions/runs/13566784339)

- [2025-03-03T19:28:44.000Z](https://github.com/pulumi/pulumi-kubernetes-operator/actions/runs/13566784339)

- [2025-03-20T12:51:10.000Z](https://github.com/pulumi/pulumi-kubernetes-operator/actions/runs/13969725750)

CI does not contain the `protoc` generator binaries:

```
make[1]: Entering directory '/home/runner/work/pulumi-kubernetes-operator/pulumi-kubernetes-operator/agent'
cd pkg/proto && protoc --go_out=. --go_opt=paths=source_relative \
    	--go-grpc_out=. --go-grpc_opt=paths=source_relative *.proto
/bin/sh: 1: protoc: not found
make[1]: *** [Makefile:6: protoc] Error [12](https://github.com/pulumi/pulumi-kubernetes-operator/actions/runs/13566784339/job/37921568924#step:10:13)7
```

--- a/.mise.toml
+++ b/.mise.toml
@@ -1,7 +1,8 @@
 [tools]
 golangci-lint = "1.58.1"
 etcd = "3.5.16"
-protoc-gen-go = "1.34.2"
+protoc = "29.3"
+protoc-gen-go = "1.36.4"
 protoc-gen-go-grpc = "1.5.1"
 kubebuilder = "4.2.0"
 envsubst = "1.4.2"

--- a/agent/Makefile
+++ b/agent/Makefile
@@ -1,6 +1,6 @@
 VERSION ?= $(shell git describe --tags --always --dirty)
 
-all: protoc agent
+all: agent
 
 protoc:
 	cd pkg/proto && protoc --go_out=. --go_opt=paths=source_relative \
@@ -9,7 +9,7 @@ protoc:
 test:
 	go test -covermode=atomic -coverprofile=coverage.out -coverpkg=./... -v ./...
 	
-agent: protoc
+agent:
 	go build -o pulumi-kubernetes-agent \
 		-ldflags "-X github.com/pulumi/pulumi-kubernetes-operator/v2/agent/version.Version=${VERSION}" \
 		main.go

